docs/gradle-plugin/configuring.md

```diff
@@ -539,6 +539,24 @@ koverReport {
 ```
 This will add the reports contents of `release` Android build variant to default reports.
 
+Is it acceptable to use wildcards `*` (zero or several arbitrary characters) and `?` (strictly one arbitrary character).
+The pattern is case-insensitive: `release` and `RelEase` processed the same way.
```

Comment:
Well, probably a bad idea. But we need to figure out how to cover with one clear rule the cases when one pattern (like `*[r]elease`) captures both `release` and `fooRelease`

Reply:
I guess writing two lines
```
mergeWith("release")
mergeWith("*Release")
```

won't harm in that particular [probably rare] case.

Reply:
This will not work, because there is a check that the rule points to at least one really existing build variant.
This is done so that the copy-pasted configurations are not clogged with patterns that do nothing.

Reply:
> And it will be satisfied in case we have release and fooRelease

Unfortunately, there is no such project in which there could be build variants `release` and `fooRelease` at the same time. 
Acceptable, for example, `fooRelease` and `barRelease`.